astroplan/conftest.py

```diff
@@ -16,3 +16,7 @@
     del PYTEST_HEADER_MODULES['h5py']
 except NameError:  # needed to support Astropy < 1.0
     pass
+
+# Mock FixedTarget.from_name class method for tests without remote data
```

Comment:
I think it would be nicer if you hide the implementation of how the mocking is done in a utility function that you call before running the tests or building the docs. Maybe `astroplan._mock_remote_data`?
Currently it would only contain this one line, but I think there will be other stuff in the future, and it gives us a nice place to put a docstring explaining what this is.
Since this is mostly for us, I wouldn't make it part of the public astroplan API.